openvpn-install.sh

```diff
@@ -1248,10 +1281,11 @@ function manageMenu () {
 	echo "What do you want to do?"
 	echo "   1) Add a new user"
 	echo "   2) Revoke existing user"
-	echo "   3) Remove OpenVPN"
-	echo "   4) Exit"
-	until [[ "$MENU_OPTION" =~ ^[1-4]$ ]]; do
-		read -rp "Select an option [1-4]: " MENU_OPTION
+	echo "   3) List Current Issued Certificates"
+	echo "   8) Remove OpenVPN"
+	echo "   9) Exit"
```

Comment:
3, 8, 9?

Reply:
I've got multiple other options that I've added to my repo And I was planning on slowly submitting all the changes up to the main repo.
And I didn't want to continually be renumbering the remove and Exit options
Such as:
        echo "   1) Add a new user"
	echo "   2) Revoke existing user"
	echo "   3) List Current Issued Certificates"
	echo "   4) List Current Active Users"
	echo "   5) Backup Configuration"
	echo "   6) Restore Configuration from Backup *Incomplete"
	echo "   7) Sync Configuration to Alternate Servers *Incomplete"
	echo "   8) Remove OpenVPN"
	echo "   9) Exit"

Reply:
Please do the appropriate changes for this PR :)

Reply:
I think I totally Screwed up my branch. I will close this pull Request. And pull another with the neccessary Changes.

Reply:
Ok :)